### PR TITLE
(WIP) Home page to React

### PR DESCRIPTION
**Pull request by thanhlmm**
## Goal
- Port current wordpress to gatsby

## Demo
https://memex-gatsby.netlify.com

## Technical changes
- I using SASS for styling

## TODO
- [ ] Add all actions for pages
- [ ] Reponsive for mobile
- [ ] Migrate legacy styles in `src/css/styles.css`

### CODE REVIEW

**Comment by thanhlmm** [comment]
There still issues when 1st time loading. The page layout will crash. I'm investigating on this

**Comment by ShishKabab** [comment]
Great work, thanks a lot!

Generally, I'd ask you please use style modules, like `styles.modules.(s)css` and import them whenever you need to use them in a React component. Using global (S)CSS make it hard to follow which style rules come from where in the project. More info here:
https://www.gatsbyjs.org/docs/css-modules/
https://www.gatsbyjs.org/tutorial/part-two/#css-modules

Also, I see you've used SCSS instead of the CSS processed with PostCSS. That indeed may be a better option, since it automatically plays nice with syntax highlighting in editors. I'll have to communicate this with the rest of the team though.

**Comment by thanhlmm** [comment]
> Great work, thanks a lot!
> 
> Generally, I'd ask you please use style modules, like `styles.modules.(s)css` and import them whenever you need to use them in a React component. Using global (S)CSS make it hard to follow which style rules come from where in the project. More info here:
> https://www.gatsbyjs.org/docs/css-modules/
> https://www.gatsbyjs.org/tutorial/part-two/#css-modules
> 

I still using css for scoping in component. You can check it there
https://github.com/WorldBrain/Website/pull/2/files#diff-363fdae0680bf309a8e7545b5435d489R1
https://github.com/WorldBrain/Website/pull/2/files#diff-ff536b9a708f1835475bf6267f009419R3

As my experience, use module css is way more adding overhead cost with very small benefit cause it hard to debug on browser. The balance is using is scope on level component. 

> Also, I see you've used SCSS instead of the CSS processed with PostCSS. That indeed may be a better option, since it automatically plays nice with syntax highlighting in editors. I'll have to communicate this with the rest of the team though.

It's good. I think we need to make some design system to use across project because it share same style guide, mood board

**Comment by ShishKabab** [comment]
> As my experience, use module css is way more adding overhead cost with very small benefit cause it hard to debug on browser. The balance is using is scope on level component.

Yeah, I understand your thinking here. For us though, with a bigger project a growing theme, local styles do make sense everywhere because it makes the code easier to follow. Looking at a component, you immediately know where all styles come from, and never have the problem of unintended conflict. I'll discuss in the team whether it's worth it to change it back, because I see the resulting styles already have a certain layout that is a bit difficult to change at this point.

For now, I think the styling looks quite good and it's probably most useful to focus on the authentication, etc.

Please follow the convention here of using the `Services` object and passing dependencies down to other components as needed, as to avoid globals and improve testability. If you have any questions about this, feel free to drop by in Slack  :)

File: src/pages/pricing/index.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ class PricingPage extends Page {
 
     this.chargebeeInstance = window['Chargebee'].init({
       site: payment.siteUrl,
+      domain: 'memex-gatsby.netlify.com'
```
Comment (blackforestboi) [inline src/pages/pricing/index.tsx]: You can now use getmemex.com here

**Comment by blackforestboi** [comment]
What makes the build here take 23minutes. Seems like a lot? 

Screenshot 2019-11-11 at 18 19 29

**Comment by blackforestboi** [comment]
Also, can you try to fix the failing test @thanhlmm because this way we can test those builds with the new netlify integration. 

BTW: which account do you want to use? is it @thanhlm27 or @thanhlmm > the last one does not have an image.

**Comment by blackforestboi** [comment]
@thanhlmm ping on the last question :)

**Comment by thanhlmm** [comment]
@blackforestboi I should be @thanhlmm 
I use 2 accounts, 1 on personal laptops, others for works

**Comment by blackforestboi** [comment]
Good to know :) 

@thanhlmm Can we get those build errors ironed out, then we can start pushing the page to netlify. also: Why does the build take so long. Seems odd for such a small page.

**Comment by thanhlmm** [comment]
@blackforestboi I gonna close this PR since it out date because we changed the design.
New PR here https://github.com/WorldBrain/Website/pull/3

> Why does the build take so long. Seems odd for such a small page.
It takes abt 4mins to build in Netlify.
